docker/scripts/dev_start.sh

```diff
@@ -84,19 +84,31 @@ EOF
 exit 0
 }
 
-function stop_containers() {
-running_containers=$(docker ps --format "{{.Names}}")
-for i in ${running_containers[*]} ; do
-  if [[ "$i" =~ apollo_* ]];then
-    printf %-*s 70 "stopping container: $i ..."
-    docker stop $i > /dev/null
-    if [ $? -eq 0 ];then
-      printf "\033[32m[DONE]\033[0m\n"
+function stop_all_apollo_containers_for_user() {
+    local force="$1"
+    local running_containers
+    running_containers="$(docker ps -a --format '{{.Names}}')"
+    for container in ${running_containers[*]} ; do
+        if [[ "${container}" =~ apollo_.*_${USER} ]] ; then
```

Comment:
Only stop and remove user-specific apollo containers. Useful when working with a multi-user workstation.